programmers/난이도별/level01.시저_암호/sjjyy.java

```diff
@@ -0,0 +1,30 @@
+package 난이도별.level01.시저_암호;
+
+public class sjjyy
+{
+    public static String solution(String s, int n) {
+
+        StringBuilder ans = new StringBuilder();
+
+        for (int i = 0 ; i < s.length() ; i++) {
+            char c = s.charAt(i);
+
+            if (Character.isLowerCase(c))
+                c = (char)('a' + (c + n - 'a') % 26);
+
+                else if (Character.isUpperCase(c))
+                    c = (char) ('A' + (c + n - 'A') % 26);
+
+                    ans.append(c);
+        }
+
+        return ans.toString();
```

Comment:
```suggestion
        return s.chars()
               .mapToObj(c -> Character.isLowerCase(c) ? (char)('a' + (c + n - 'a') % 26) :
                              Character.isUpperCase(c) ? (char)('A' + (c + n - 'A') % 26) :
                              c)
               .collect(Collectors.joining(""));
```
이렇게 표현할 수 있을 것 같습니다!

Reply:
collect를 사용할 수 없다고 해서 좀 더 찾아보겠습니다!

Reply:
항상 코드 보면서 많이 배워요 감사합니다